### ISSUE TITLE
Merge CLI repo to YoMo repo

### ISSUE THREAD

**Issue by fanweixiao** [yomorun/yomo#361]
### Summary

As getting off go plugin, there is no need to separate YoMo and CLI to different repos, should merge them together, and change install script.

Release v1.9.0 for this changes, no breaking changes.

**Comment by gedw99** [comment]
any updates this ? there seems to be overlap in the code base so its a bit hard to follow.

**Comment by fanweixiao** [comment]
Hi @gedw99, already merged, please upgrade to newer version.
